android/app/build.gradle

```diff
@@ -216,9 +217,11 @@ dependencies {
     implementation project(':react-native-securerandom')
     implementation project(':react-native-webview-bridge')
     implementation project(':react-native-config')
-    implementation project(':react-native-fcm')
-    implementation 'com.google.firebase:firebase-core:10.0.1' //this decides your firebase SDK version
-    compile project(':react-native-testfairy')
+    implementation 'com.google.android.gms:play-services-base:15.0.0'
+    implementation 'com.google.firebase:firebase-core:15.0.2' //this decides your firebase SDK version
+    implementation "com.google.firebase:firebase-messaging:15.0.2"
+    implementation 'me.leolin:ShortcutBadger:1.1.21@aar'
+    implementation project(':react-native-testfairy')
```

Comment:
Please comment why we need all those.

Reply:
![image](https://user-images.githubusercontent.com/13368542/40554787-b790b63e-5ffb-11e8-9e71-82350d3075ee.png)

Reply:
![image](https://user-images.githubusercontent.com/13368542/40554807-c4bd41f6-5ffb-11e8-88e1-8e062c5c80c6.png)

Reply:
badges? Don't think we want that.

Reply:
okay @jeluard, will revert that as well.